mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.h

```diff
@@ -55,6 +55,33 @@ enum class SparseEmitStrategy {
   kDebugInterface, // generate only place-holder for sparse iteration
 };
 
+namespace sparse_tensor {
+/// Select between different loop ordering strategies.
```

Comment:
Selects between (style in this module is that method comments use the s-form for verbs)